src/member/member.ts

```diff
@@ -52,22 +53,53 @@ export const buildMemberExtra = (extra: Partial<MemberExtra>) => ({
   ...extra,
 });
 
-export type Member = {
+export type Account = {
   id: UUID;
   name: string;
-  email: string;
 };
 
-export type CompleteMember = Member & {
+type AccountTypeProperty = {
   type: `${MemberType}` | MemberType;
```

Comment:
In this way, we avoid using the term `Account` and `Member` interchangeably, as it feels to me when I see a type `AccountTypeProperty` that use type of `MemberType`. I don't know what do you think ?

 ```suggestion
  type: `${AccountType}` | AccountType;
```